docs/src/quickstart/hello_klio_streaming.rst

```diff
@@ -28,13 +33,17 @@ Create a New Klio Job
 
     $ klio job create --job-name klio-quick-start --create-resources --use-defaults
 
+
 After responding to the prompts, Klio will:
 
-1. Create two Pub/Sub topics, one for input and one for output, in the provided GCP project: ``projects/$GCP_PROJECT/topics/klio-quick-start-input``  and ``projects/$GCP_PROJECT/topics/klio-quick-start-output``.
-2. Create one Pub/Sub subscription to the input Pub/Sub topic in the provided GCP project: ``projects/$GCP_PROJECT/subscription/klio-quick-start-input-klio-quickstart-input``.
-3. Create a GCS bucket in the provided GCP project you provided in the prompt for output data: ``gs://$GCP_PROJECT-output``.
-4. Create a `Google Stackdriver`_ dashboard in the provided GCP project for you to monitor runtime job metrics.
-5. Create required files within the current working directory.
+1. Create a GCS bucket in the provided GCP project you provided in the prompt for output data: ``gs://$GCP_PROJECT-output``.
+2. Create a `Google Stackdriver`_ dashboard in the provided GCP project for you to monitor runtime job metrics.
+3. Create required files within the current working directory.
+4. Create two Pub/Sub topics, one for input and one for output,
+    in the provided GCP project: ``projects/$GCP_PROJECT/topics/klio-quick-start-input``
+    and ``projects/$GCP_PROJECT/topics/klio-quick-start-output``.
+5. Create one Pub/Sub subscription to the input Pub/Sub topic
+   in the provided GCP project: ``projects/$GCP_PROJECT/subscription/klio-quick-start-input-klio-quickstart-input``.
```

Comment:
This line and the lines in number 4 above, because they start with a tab, are rendered weirdly. I think the only thing we can do to render it correctly is have no new lines; it's all just on one line.